contentcuration/contentcuration/frontend/channelEdit/components/ResourcePanel.vue

```diff
@@ -554,6 +605,89 @@
       getText(field) {
         return this.node[field] || this.defaultText;
       },
+      matchIdToString(ids) {
+        return ids.map(i => this.translateMetadataString(camelCase(i))).join(', ');
```

Comment:
So...translate each string in the array, and then pass the current language of the UI and the translated array into the list formatter?

Reply:
Yes.

Reply:
After reviewing the figma, it looks like using a conjunction is not what is in the design (just commas)... I did try out adding the list format and it worked so I can go back and add this in quite easily, but I did not commit those changes because I'm not sure what the benefit would be over doing a join

Reply:
The benefit over doing a `join(', ')` is that it will internationalize the listing, depending on locale. Not every language uses commas to generate lists in the way that English does, and it will ensure that there are no gotchas for RTL languages.

Please also rename the function to something like `metadataListText` or somesuch to make it more descriptive!